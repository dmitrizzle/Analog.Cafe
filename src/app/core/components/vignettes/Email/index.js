import React from "react"

import Link from "../../controls/Link"

export const emailString = "d@analog.cafe"
export default () => {
  return (
    <strong>
      <a href="mailto:d@analog.cafe">email</a>
    </strong>
  )
}
