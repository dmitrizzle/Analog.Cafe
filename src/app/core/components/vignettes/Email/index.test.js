import "jest-styled-components"

import { shallow } from "enzyme"
import React from "react"

import Email from "./"

it("Render Email, matches snapshot", () => {
  const element = shallow(<Email />)
  expect(element).toMatchSnapshot()
})
