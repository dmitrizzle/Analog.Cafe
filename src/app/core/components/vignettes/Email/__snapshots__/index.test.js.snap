// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Email, matches snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <a
        href="mailto:d@analog.cafe"
>
        email
</a>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "email",
        "href": "mailto:d@analog.cafe",
      },
      "ref": null,
      "rendered": "email",
      "type": "a",
    },
    "type": "strong",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <a
          href="mailto:d@analog.cafe"
>
          email
</a>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "email",
          "href": "mailto:d@analog.cafe",
        },
        "ref": null,
        "rendered": "email",
        "type": "a",
      },
      "type": "strong",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
